### ISSUE TITLE
The moksha desktop tries to show non-existant file when used on another distro

### ISSUE THREAD

**Issue by texstar** [JeffHoogland/moksha#102]
I'm not sure if Moksha 0.3.1 is just for Bodhi Linux but it tries to open this file that does not exist on PCLinuxOS.
  
 snprintf(buff, sizeof(buff), "enlightenment_open "
            "file:///usr/share/bodhi/quickstart/quickstartEN/index.html");

Should check to see if the file exists before trying to open it in a browser. 

located in moksha-0.3.1/src/bin/e_utils.c

**Comment by rbtylee** [comment]
Yep sure does. That is kinda an undocumented dependency. You can get it [here](http://packages.bodhilinux.com/bodhi/pool/b5main/b/bodhi-quickstart/).

Moksha also uses [Moksha menu](https://github.com/BodhiDev/Moksha-menu). Another undocumented here dependency. Plus has configurations settings set for using a certain gtk theme and icon set. Look thru the code.

I prob should document all this but that has been low priority.
